app/controllers/hubstats/pull_requests_controller.rb

```diff
@@ -33,6 +33,7 @@ def show
       @pull_request = Hubstats::PullRequest.belonging_to_repo(@repo.id).where(id: params[:id]).first
       @comments = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).limit(20)
       @comment_count = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).count(:all)
+      @deploys = Hubstats::Deploy.where(id: @pull_request.deploy_id)
```

Comment:
A pull request can only belong to one deploy, so a length doesn't really make sense, does it?

A possible workaround that still makes use of our association:
```
@deploy = [ @pull_request.deploy ]
```